src/QsCompiler/CompilationManager/CompilationUnit.cs

```diff
@@ -74,6 +77,42 @@ public class Headers
             attributes.Select(IsDeclaration("TypeDeclarationAttribute")).Where(v => v != null)
                 .Select(TypeDeclarationHeader.FromJson).Select(built => built.Item2);
 
+        /// <summary>
+        /// Renames all private and internal declarations in the headers to have names that are based on the ID of the
+        /// referenced assembly.
+        /// </summary>
+        /// <param name="id">A number uniquely identifying the referenced assembly that contains the headers.</param>
+        /// <param name="source">The path to the assembly that is the source of the given headers.</param>
+        /// <param name="headers">The declaration headers in the assembly.</param>
+        /// <returns></returns>
+        private static Headers RenameInternals(int id, string source, Headers headers)
+        {
+            static bool IsInternal(AccessModifier access) => access.IsInternal || access.IsPrivate;
```

Comment:
I assume the IsPrivate is removed with the next PR, so I don't need to suggest picking another name. :)

Reply:
Yes, the name is not great... Depending on which PR is merged first I will either update this PR or the remove private PR, so that this function is removed.